docs/content/configuration-and-management/group-membership-known-issues.md

```diff
@@ -0,0 +1,161 @@
+- The Service Account continues to exist in the tier namespace.
+- Kubernetes TokenReview validates the Service Account, not the original user's group membership.
+
+**Example Scenario**:
+
+```text
+T+0h:   User "alice" is in "premium-users" group
+T+0h:   Alice requests a token -> Gets SA token in maas-default-gateway-tier-premium namespace
+T+1h:   Admin removes Alice from "premium-users" group
+T+1h:   Alice's token is STILL VALID (expires at T+24h)
+T+1h:   Alice can still make requests using the existing token
+T+24h:  Token expires, Alice must request a new one
+T+24h:  New token request -> Alice gets "free" tier (or fails if no tier matches)
+```
+
+**Workaround**:
+
+- Revoke the user's tokens explicitly using the `DELETE /v1/tokens` endpoint.
+- This deletes and recreates the user's Service Account, invalidating all existing tokens.
+
+```bash
+curl -X DELETE "${HOST}/maas-api/v1/tokens" \
+  -H "Authorization: Bearer ${USER_TOKEN}"
+```
+
+Note: The user must authenticate with their own token to revoke their tokens. Administrators cannot revoke tokens on behalf of other users in the current implementation.
+
+#### 2. Rate Limiting Continues at Old Tier
+
+**Impact**: Medium
+
+**Description**:
+
+The AuthPolicy caches the tier lookup result (default TTL: 5 minutes). After a user is removed from a group:
+
+- Requests within the cache window continue to use the old tier's rate limits.
+- After cache expiry, the tier is re-evaluated based on current group membership.
+- If the user still has a valid token but no longer belongs to any tier group, requests may fail.
+
+**Example Timeline**:
+
+```text
+T+0m:   User removed from "premium-users" group
+T+1m:   Request made -> Cached tier "premium" used -> Rate limit: 1000 tokens/min
+T+5m:   Cache expires
+T+6m:   Request made -> Tier lookup fails (no matching group) -> Request may fail with 403
+```
+
+**Workaround**:
+
+- Wait for cache TTL (5 minutes) for rate limiting to reflect the new group membership.
+- For immediate effect, restart Authorino pods (disruptive).
+
+#### 3. Service Account Persists After Group Removal
+
+**Impact**: Low
+
+**Description**:
+
+When a user is removed from a group, their Service Account in the tier namespace is not automatically deleted:
+
+- The Service Account remains in the tier namespace.
+- No new tokens can be issued for the old tier (tier lookup fails).
+- Old tokens continue to work until expiration.
+- This is a cleanup artifact, not a security issue (access is controlled by RBAC and rate limiting).
+
+**Workaround**:
+
+- Service Accounts can be manually cleaned up if needed:
+
+```bash
+kubectl delete serviceaccount <username-hash> -n maas-default-gateway-tier-<old-tier>
+```
+
+#### 4. User Downgrade Creates New Service Account
+
+**Impact**: Low
+
+**Description**:
+
+When a user is moved to a lower tier (e.g., removed from `premium-users`, now only in `system:authenticated`):
```

Comment:
I think `system:authenticated` is our sample setup, not a hard rule. Maybe we can clarify this.